docs/pages/setup/guides/docker.mdx

```diff
@@ -9,13 +9,17 @@ This section will cover:
 - Getting started with a local Teleport using Docker.
 - Using Teleport with Teleport's native client, `tsh`.
 
+<Admonition scope={["oss", "ent"]} type="note" title="Enterprise using Docker" scopeOnly={true}>
```

Comment:
Admonitions don't currently support the `scope` attribute. Since this is a pretty short admonition, I think it would look okay as a `Notice`:

```tsx
<Notice scope={["oss", "enterprise"]} type="note">
To run Teleport Enterprise using Docker we recommend following 
this guide:

[Run Teleport Enterprise using Docker](../../enterprise/getting-started.mdx#run-teleport-enterprise-using-docker)
</Notice>
```

Reply:
Also, should this have the `cloud` scope instead of `oss`?

Reply:
@ptgott changed to notice.  This notice only applies to oss and enterprise since if you want to use enterprise with docker they should use another guide